src/routes/landingPage/landingPage.jsx

```diff
@@ -0,0 +1,34 @@
+import "./landingPage.css";
+import poster from "../../assets/undraw_add_notes_re_ln36.svg";
+export const LandingPage = () => {
+  return (
+    <div className="grid-container box-shadow">
+      <div className="details-page-container">
+        <div className="header-container">
+          <h1 className="text-align-left">
+            <span className="primary-color">Pocket</span> Notes
+          </h1>
+        </div>
+        <div className="details-container flx-col text-align-left">
+          <h3>Meet your modern</h3>
+          <h3 className="primary-color">Note Taking App</h3>
+          <p>
+            Manage your daily tasks and workflow in a modern way and boost your
+            efficiency without any efforts.
+          </p>
+        </div>
+        <div className="authentication-container flex-col text-align-left">
+          <button className="button-style-none solid-button disable-hover button-style">
+            Join Now
+          </button>
+          <a className="login-link" href="">
```

Comment:
use LINK element instead of anchor tag

Reply:
Sure, i will add this after adding React-router.